setup.py

```diff
@@ -87,7 +112,9 @@ def run(self):
         "oss": oss,
         "ssh": ssh,
         # NOTE: https://github.com/inveniosoftware/troubleshooting/issues/1
-        ':python_version=="2.7"': ["futures", "pathlib2"],
+        ":python_version=='2.7'": ["futures", "pathlib2"],
+        ":python_version>='3.6'": ["black==19.3b0"],
```

Comment:
Let's not include `black` into regular dependencies. It should be a part of test_requirements. Let's do something like:

```
if sys.version_info >= (3, 6):
    test_requires.append('black==19.3b0')
```